Views/ProductTypes/Buy.cshtml

```diff
@@ -3,12 +3,21 @@
 
 <h2>Product Types</h2>
 
-<a asp-controller="Products"><input value="View All Products" type="button" class="btn btn-primary"/></a>
-
+<a asp-controller="Products"><input value="View All Products" type="button" class="btn btn-primary btn-sm"/></a>
+<table class="table table-striped">
+    <thead>
+        <tr>
+            <th>Product Category</th>
+            <th>Amount of Products</th>
+        </tr>
+    </thead>
+    <tbody>
 @foreach (ProductType type in Model.ProductTypes)
 {
-  <div class="row">
-      <div class="col-md-3"><a asp-action="List" asp-route-id="@type.ProductTypeId">@Html.DisplayFor(model => type.Label)</a></div>
-      <div class="col-md-1">(@Html.DisplayFor(model => type.Quantity))</div>
-  </div>
-}
\ No newline at end of file
+  <tr>
+      <td><a asp-action="List" asp-route-id="@type.ProductTypeId">@Html.DisplayFor(model => type.Label)</a></td>
+      <td>(@Html.DisplayFor(model => type.Quantity))</td>
+  </tr>
+}
+    </tbody>
+</table>
```

Comment:
Apparently old school tech messes up without an empty line character. http://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-file

But everything LGTM